packages/venia-ui/lib/components/CategoryList/categoryTile.css

```diff
@@ -15,6 +15,7 @@
     display: block;
     height: 5rem;
     object-fit: cover;
+    object-position: center;
```

Comment:
This isn't strictly necessary I suppose, the initial value of `50% 50%` will have the same result.

Reply:
Unless you think it's better for readability (ie. the common developer doesn't know centering is the default), let's remove it. Not going to block my approval though.